backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/codec/impl/AbstractElasticsearchJavaTimeFieldCodec.java

```diff
@@ -60,6 +60,11 @@ public T decodeAggregationKey(JsonElement key, JsonElement keyAsString) {
 		return decode( keyAsString );
 	}
 
+	@Override
+	public T convertFromDoubleNullSafe(Double value) {
+		return null;
```

Comment:
Have you tried date fields with the Elasticsearch backend? Is it not possible to do any metric aggregations with them? if not, this probably should fail with an assertion error and corresponding aggregations just not added to the `ElasticsearchIndexValueFieldType.Builder`...